test/streamparse/test_streamparse_run.py

```diff
@@ -0,0 +1,42 @@
+"""
+Tests for streamparse_run
+"""
+
+import logging
+import os
+import sys
+import unittest
+
+from streamparse import run
+
+
+class StreamparseRunTests(unittest.TestCase):
+
+    run_target_invoked = False
+    run_target_invoked_serializer = None
+
+    def test_streamparse_run_storm_1_0_2(self):
+        # patch streamparse_run sys argv assuming tests run
+        # from streamparse project root, (Storm <= 1.0.2 case)
+        sys.argv = ['streamparse_run',
+                    'test.'
```

Comment:
Yeah, I'd prefer just having the string all on one line, or doing:

```py
sys.argv = ['streamparse_run',
            ('test.'
             'streamparse_run.streamparse_run_target.StreamparseRunTarget')]
```

to make it more obvious.